tilequeue/store.py

```diff
@@ -36,6 +36,9 @@ class KeyFormatType(Enum):
     # prefix comes before hash
     prefix_hash = 2
 
+    # hash omitted
+    no_hash = 3
```

Comment:
to follow the convention maybe it shall be just `prefix` ?

Reply:
nm, I went with your suggestion